src/components/TalkToUs/style.js

```diff
@@ -133,3 +107,29 @@ export const menuItem = theme => css`
 export const menuItemImage = css`
   margin-right: 1.6rem;
 `
+
+export const container = (theme, closing) => css`
+  position: fixed;
+  height: 55rem;
+  width: 34rem;
+  background-color: white;
+  bottom: 1.6rem;
+  right: 1.6rem;
+  border-top-left-radius: 1rem;
+  border-top-right-radius: 1rem;
+  box-shadow: 0 0 1rem #000000;
+  box-sizing: border-box;
+  z-index: ${theme.zIndices.content};
+  transition-property: transform, opacity;
+  transform: scale(1);
+  opacity: 1;
+  transition-duration: 0.3s;
+  transition-timing-function: linear;
+
+  ${mq.mobile} {
+    width: calc(100% - 2.4rem);
+    max-width: 32rem;
+  }
+
+  ${closing && `transform:translate(40%, 40%) scale(0.001); opacity: 0`}
```

Comment:
transition related style can be simplified as
```
transform: ${closing ? "translate(40%, 40%) scale(0.001)" : "scale(1)"};
```

Reply:
fixed.